lighthouse-cli/test/smokehouse/core-tests.js

```diff
@@ -72,8 +72,7 @@ const smokeTests = [
   errorsExpiredSsl,
   errorsIframeExpiredSsl,
   errorsInfiniteLoop,
-  // TODO: restore when --enable-features=AutofillShowTypePredictions is not needed.
-  // formsAutoComplete,
+  formsAutoComplete,
```

Comment:
I've enabled the smoke test, but commented out the stuff that rely on the predictions being available.